README.adoc

```diff
@@ -7,9 +7,21 @@ image:https://img.shields.io/badge/Angular%20Version-4-green.svg[]
 image:https://badges.frapsoft.com/typescript/code/typescript.svg?v=101[]
 image:https://circleci.com/gh/aerogear/cordova-showcase-template.svg?&style=shield[link=https://circleci.com/gh/aerogear/cordova-showcase-template]
 
+[width="85%"]
+|===========================================================================================================
+|                 | *Project Info*                                                                            
+| License:        | Apache License, Version 2.0                                                             
+| Build:          | NPM                                                                                     
+| End User Documentation:  | link:https://docs.aerogear.org[]                                                 
+| Community Documentation: | link:https://www.aerogear.org[]                                                      
+| Issue tracker:  | link:https://issues.jboss.org/browse/AEROGEAR[] 
+| Mailing lists:  | link:https://groups.google.com/forum/#!forum/aerogear[aerogear-dev] 
+|===========================================================================================================
+
 A template application to showcase features from link:https://github.com/aerogear/aerogear-js-sdk[Aerogear Js SDK].
 
 As this application showcases features from link:https://github.com/aerogear/aerogear-js-sdk[Aerogear Js SDK] ensure you checkout the desired git tag for whatever SDK version you intend to run. Check link:https://www.npmjs.com/org/aerogear[NPM] for associated versions and releases.
+
 == Build Instructions
```

Comment:
can we add a pre-req section and list all the tools/IDEs/SDKs developer need to install